repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/AtomEntryStreamBuilder.scala

```diff
@@ -112,6 +132,10 @@ object AtomEntryStreamBuilder {
 
   case class UnrecoverableIOException(cause: Throwable) extends RuntimeException(cause)
 
+  case class ServerResponseException(message: String) extends Exception(message)
+
+  object ClientErrorException extends Exception
+
   object AuthenticationException extends Exception
```

Comment:
It was late in the day and I was fading. Full `inner classes` not needed, just `case class`es instead of `objects`. C/W. Pushing now.